src/scripts/create-user.ts

```diff
@@ -0,0 +1,133 @@
+import { CreateUserRequestOptions } from '@okta/okta-sdk-nodejs';
+import { Argv } from 'yargs';
+import { RootCommand } from '..';
+
+import {
+  oktaManageClient,
+  OktaConfiguration,
+  User,
+  oktaUserAsUser,
+  getUser,
+} from './services/user-service';
+
+import { generatePassword } from './services/password_manager';
+
+const createUser = async (
+  oktaConfiguration: OktaConfiguration,
+  password: string,
+  email: string,
+  firstName = 'Unknown',
```

Comment:
Prefer not to default these - we don't want Okta full of "Unknown User".

Reply:
The optional parameters were legacy code, they have been removed